frontend/utils/fetcher.ts

```diff
@@ -66,6 +66,7 @@ export const getErrorMessage = async (res: Response) => {
     messageError = `${res.statusText}: ${(await res.json()).error.message}`
   } catch (e) {
     // unable to identify error message, possibly a network failure
+    return 'Unknown error - Please contact Bailo support'
```

Comment:
Is it possible to put a link directly to our help page here?

Reply:
It would be nice to, but this is just a util function that returns a string. We'd need additional logic everywhere it's being used to route the user to that page in the event of this happening.

I'm not against us doing that, but this isn't strictly a part of this ticket. I just noticed we weren't providing any feedback in this very niche scenario, so figured I'd put at least something in.

We can raise a ticket to address this if you want?